src/zenml/cli/user_management.py

```diff
@@ -191,11 +205,30 @@ def create_user(
     required=False,
     help="New user email.",
 )
+@click.option(
+    "--password",
+    "-p",
+    "updated_password",
+    type=str,
+    required=False,
+    help="New user password.",
+)
```

Comment:
So nice that you decided to add this here. Someone was recently asking for this.